src/controllers/resources/form/products.js

```diff
@@ -2135,5 +2135,8 @@ export default function () {
     setTimeout(() => {
       listOrders(tabId)
     }, 1500)
+    document.getElementById(`t${tabId}-ad-relevance-link`).addEventListener('click', () => {
+      document.getElementById(`t${tabId}-ad-relevance`).classList.remove('d-none')
+    })
```

Comment:
```suggestion
    }, false)
```

Garantir `useCapture` false, perf